src/App.tsx

```diff
@@ -15,48 +16,117 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  Alphabetically = 'isAlphabeticaly',
+  Length = 'isLength',
+}
+
+type Props = {};
+
+export const App: React.FC<Props> = () => {
```

Comment:
```suggestion

export const App: React.FC = () => {
```
There's no need to specify props if you don't have it.